tests/construction/dj_query_test.py

```diff
@@ -28,45 +27,44 @@ async def test_build_dj_metric_query(mocker, request):
  FROM (SELECT  SUM(basic_DOT_transform_DOT_country_agg.num_users) AS sum_users,
         basic_DOT_transform_DOT_country_agg.country AS a_DOT_country,
         basic_DOT_transform_DOT_country_agg.num_users AS a_DOT_num_users,
-        b.id AS b_DOT_id,
-        b.full_name AS b_DOT_full_name,
-        b.names_map AS b_DOT_names_map,
-        b.user_metadata AS b_DOT_user_metadata,
-        b.age AS b_DOT_age,
-        b.country AS b_DOT_country,
-        b.gender AS b_DOT_gender,
-        b.preferred_language AS b_DOT_preferred_language,
-        b.secret_number AS b_DOT_secret_number
- FROM (SELECT  basic.comments.country,
-        COUNT(DISTINCT basic.comments.id) AS num_users
- FROM basic.comments
+        basic_DOT_source_DOT_users.id AS b_DOT_id,
```

Comment:
If we show these `_DOT_`s to some users, they'll think we are insane ;)

Reply:
Agreed, I'll add this to the sync to discuss the best syntax for these system generated aliases.